cmd/kube-apiserver/app/config.go

```diff
@@ -86,6 +90,17 @@ func NewConfig(opts options.CompletedOptions) (*Config, error) {
 		return nil, err
 	}
 
+	if ocfg := enablement.OpenshiftConfig(); ocfg != nil {
+		// Add MinimumKubeletVerison authorizer, to block a node from being able to access most resources if it's not new enough.
+		// We must do so here instead of in pkg/apiserver because it relies on a node informer, which is not present in generic control planes.
+		genericConfig.Authorization.Authorizer = union.New(genericConfig.Authorization.Authorizer, minimumkubeletversion.NewMinimumKubeletVersion(
```

Comment:
Have you considered adding the new authorizer to [the authorizer config](https://github.com/openshift/kubernetes/commit/72a6eb500782a31b3f7246596d00ddba4a0b6a8d#diff-6c756a18b342480afacdf8ea8b4e150656d3117b06136f449c08b8047ef0d4aeR128) ?

It has access to the node informer (`config.VersionedInformerFactory.Core().V1().Nodes()`) and can also read the `enablement.OpenshiftConfig`.

Also, the list of enabled authorizers is [passed directly from the operator](https://github.com/openshift/cluster-kube-apiserver-operator/blob/60392c12f52742ea6f9df605250b53287c2c24e8/bindata/assets/config/defaultconfig.yaml#L37) and is read in the config I mentioned above.

By adding the new authorizer to `authorization-mode`, we would extend the current mechanism. All authorizers would be in the same place, and it could be disabled by an admin if needed. WDYT?